synapse/tests/test_lib_aha.py

```diff
@@ -1406,3 +1467,105 @@ async def sleep99(cell):
             self.false(await cell00.killTask(task01))
 
             self.none(await cell00.getAhaProxy(feats=(('newp', 9),)))
+
+    async def test_aha_req_proxy(self):
+
+        async with self.getTestAha() as aha:
+
+            info = {
+                'urlinfo': {
+                    'host': 'localhost',
+                    'port': 12345,
+                    'scheme': 'ssl',
+                    'user': 'root',
+                },
+                'online': 'fakesession',
+                'ready': True,
+            }
+
+            await aha.addAhaSvc('testsvc.aha', info)
+            svcdef = await aha.getAhaSvc('testsvc.aha')
+            with self.raises(s_exc.NotReady):
+                await aha.reqAhaSvcProxy(svcdef)
+
+    async def test_lib_aha_peer_api(self):
+
+        async with self.getTestAha() as aha:
+
+            purl00 = await aha.addAhaSvcProv('0.cell')
+            purl01 = await aha.addAhaSvcProv('1.cell', provinfo={'mirror': '0.cell'})
+
+            cell00 = await aha.enter_context(self.getTestCell(conf={'aha:provision': purl00}))
+            cell01 = await aha.enter_context(self.getTestCell(conf={'aha:provision': purl01}))
+
+            await cell01.sync()
+
+            todo = s_common.todo('getCellInfo')
+            items = [item async for item in cell00.callPeerApi(todo)]
+            self.len(1, items)
+
+    async def test_lib_aha_peer_genr(self):
+
+        async with self.getTestAha() as aha:
+
+            purl00 = await aha.addAhaSvcProv('0.cell')
+            purl01 = await aha.addAhaSvcProv('1.cell', provinfo={'mirror': '0.cell'})
+
+            cell00 = await aha.enter_context(self.getTestCell(conf={'aha:provision': purl00}))
+            cell01 = await aha.enter_context(self.getTestCell(conf={'aha:provision': purl01}))
+
+            await cell01.sync()
+
+            todo = s_common.todo('getNexusChanges', 0, wait=False)
+            items = [item async for item in cell00.callPeerGenr(todo)]
```

Comment:
Does callPeerGenr not skip itself like callPeerApi?

Reply:
It does but the test is not exposing it very well.  The `getNexusChanges` is starting at offs=0 and there are two offs/msgs for the non-skipped cell. I'll change the test to cast the items to dict for better assertion against the one. Up in the next commit.